### DISCUSSION TITLE
Add maxFontSizeMultiplier to Options (specifically tab bar badge)

### DISCUSSION THREAD

**Discussion by dubbajubba** [react-navigation/react-navigation#11486] [Ideas]
Add a maxFontSizeMultiplier property to React Navigation components.

React Native provides a maxFontSizeMultiplier property on their Text and TextInput components: https://reactnative.dev/docs/next/text#maxfontsizemultiplier

It would be useful if this property was also available on React Navigation components. I am currently working on an app and have been testing with a user that has a high font size set on their device. I am able to set the maxFontSizeMultiplier across all text and text inputs in my app, but cannot set it on the React Navigation components. Of particular concern is the badge on the tab navigation. Here are several images from my app, starting with the default iOS font size, and going up one size per screenshot.
<img width="113" alt="image" src="https://github.com/react-navigation/react-navigation/assets/33437206/90f2a2a6-1db5-4df7-afa8-f8cf69e01ba0">
<img width="89" alt="image" src="https://github.com/react-navigation/react-navigation/assets/33437206/d5d615d8-06db-4dd0-8ede-a0c4aa63e5bd">
<img width="90" alt="image" src="https://github.com/react-navigation/react-navigation/assets/33437206/be88b3ba-cdd2-4e62-961b-f5b70c0b495d">
<img width="100" alt="image" src="https://github.com/react-navigation/react-navigation/assets/33437206/7ed1d902-e222-431c-bafe-085fd2860fea">

And this is what it looks like on my test user's device (they like very large fonts):
<img width="92" alt="image" src="https://github.com/react-navigation/react-navigation/assets/33437206/56ce872e-d627-4f33-b389-7cc4a43f69ae">

**Comment by groverParth** [comment]
I am also facing the same issue. @austin-triiistudio Did you find any solution for this ??
